daemon/client/client.go

```diff
@@ -135,7 +135,7 @@ func main() {
 	}
 
 	install()
-	install()
+	//install()
```

Comment:
why this change?

Reply:
only test i forgot to leave  the comment.